sros2/sros2/verb/generate_artifacts.py

```diff
@@ -31,11 +31,13 @@ class GenerateArtifactsVerb(VerbExtension):
     """Generate keys and permission files from a list of identities and policy files."""
 
     def add_arguments(self, parser, cli_name):
-        arg = parser.add_argument('-k', '--keystore-root-path', help='root path of keystore')
+        arg = parser.add_argument(
+            '-k', '--keystore-root-path',
+            help='root path of keystore')
         arg.completer = DirectoriesCompleter()
         parser.add_argument(
-            '-n', '--node-names', nargs='*', default=[],
-            help='list of identities, aka ROS node names')
+            '-c', '--contexts', nargs='*', default=[],
+            help='list of identities, aka ROS context paths')
```

Comment:
we could be more explicit, and use `security context` instead of `context`.

Reply:
Ok, `--security-contexts` could work, matching the rcl args.

Reply:
I was referring to the help comment, but updating the long flag version sounds good too.

Reply:
Solved in https://github.com/ros2/sros2/pull/177/commits/d6b2936845f69c66d1b6347c7783f82bc5379384